windows_containers/windows-containers-support.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="windows-containers-support"]
+= Getting support
+include::_attributes/common-attributes.adoc[]
+:context: understanding-windows-container-workloads
+
+toc::[]
+
+[id="getting-support"]
+== Getting support
+
+// wording taken and modified from https://access.redhat.com/support/policy/updates/openshift#windows
+
+Windows Container Support for {product-title} is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
```

Comment:
I know you didn't edit any of this text, so you can consider this suggestion to be out of scope: use a non-breaking space between "Red" and "Hat". See https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces.